LongRunningIntegrationTestDs3/LongRunningIntegrationTestDs3.cs

```diff
@@ -221,7 +284,7 @@ public override int Read(byte[] buffer, int offset, int count)
 
         public override void Write(byte[] buffer, int offset, int count)
         {
-            throw new NotSupportedException();
+            //throw new NotSupportedException();
```

Comment:
We do not want to write the files to disk when using in Tests, i'll remove this and add new comment